ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/CodeDelegationProcessor.java

```diff
@@ -89,6 +94,22 @@ private void processAuthorization(
       return;
     }
 
+    if (codeDelegation.nonce() == MAX_NONCE) {
```

Comment:
```suggestion
    if (codeDelegation.nonce() >= MAX_NONCE) {
```
Would make more sense to me, given the trace log this causes

Reply:
That check is a bit confusing, but is correct. `nonce` is a `long` and `MAX_NONCE` (which is 2^64-1) is the maximum value that a `long` can be. So `nonce` can never be greater than `MAX_NONCE`, because it would overflow to 0 in that case.